src/gui/pedalboard/cables/CableView.cpp

```diff
@@ -99,9 +124,26 @@ void CableView::mouseUp (const MouseEvent& e)
 
 void CableView::timerCallback()
 {
-    const auto mousePos = Desktop::getMousePosition() - getScreenPosition();
-    nearestPort = portLocationHelper->getNearestPort (mousePos, board);
-    repaint();
+    if (MouseOverClickablePort())
+    {
+        repaint();
```

Comment:
I wonder if we can avoid repainting the whole view here, like do something like `repaint (boundsAroundPort);`?

Reply:
Yeah good idea!